Learning_test_automation/Projekt#1/TeachersGradebook/SchoolActivity/SchoolGradebook.cs

```diff
@@ -141,19 +138,16 @@ private void PrintSubjectGrades(List<Grade> grades)
             }       
         }
         
-        private void PrintSubjectAverage(List<Grade> grades)
+        private void PrintSubjectAverage(List<IGrade> grades)
         {
             int index = 0;
             int totalCount = grades.Count;
-            List<double> gradeAVG = new List<double>();
 
             if (totalCount == 0) Console.WriteLine("\nYour list is empty, You add the first item to your list.");
             else
             {
-                foreach (Grade grade in grades)
-                {
-                    gradeAVG.Add(grade.Value);
-                }
+                List<double> gradeAVG = grades.Select(grade => grade.Value).ToList();
```

Comment:
Zmienilbym nazwe `gradeAVG`, zeby nie uzywac skrotu

Reply:
A ja myślałem że to takie sprytne zagranie.